engine/src/hopeit/server/web.py

```diff
@@ -648,22 +647,23 @@ async def _handle_multipart_invocation(
     except Exception as e:  # pylint: disable=broad-except
         return _failed_response(context, e)
 
-def _start_streams(app_engine: AppEngine):
+async def _start_streams(app: web.Application):
     """
     Start all stream event types configured in app.
 
-    :param app_engine: already started instance of AppEngine
-    """
-    for event_name, event_info in app_engine.effective_events.items():
-        if event_info.type == EventType.STREAM:
-            assert event_info.read_stream
-            logger.info(
-                __name__, f"STREAM start event_name={event_name} read_stream={event_info.read_stream.name}")
-            asyncio.create_task(app_engine.read_stream(event_name=event_name))
```

Comment:
Before introducing so many changes an couple stream start to aiohttp startup hook, can we diagnose what was causing the streams not to start using regular asyncio.createtask? Was this code reachable and actually starting the streams? Anything in the logs?

Reply:
The issue looks like run an asyncio.create_task(..) inside of the loop.run_until_complete() call won't start properly the task. Running the simple_example app you can observe no streams or services are listening after startup and no logs about `Starting reading stream...`, you need to start each one with the management API, and then works fine.

Reading [this topic on aiohttp docs](https://docs.aiohttp.org/en/stable/web_advanced.html?highlight=on_startup%C2%B6#background-tasks) I found the strategy I propose in the pr, it shows how to add create_task to a running web.Application.

The current commit now starts the streams and services if you use --start-streams on the startup arguments.
And logs now show the usual behavior  `Starting reading stream...`.